r/src/get_metadata_information.r

```diff
@@ -33,7 +36,11 @@ getMitochondrialContent <- function(req) {
 
     # Subset the percent.mt ordering by cells_id (DESC)
     result <- data@meta.data[order(data$cells_id, decreasing = F), "percent.mt"]
-
+    result<- as.data.frame(result)
+    result$cells_id <- data@meta.data$cells_id
```

Comment:
I cannot suggest code which has not been changed. Could you do the same thing that I suggest in the previous comment?

Another possibility is to reorder the cells in line 40 `result$cells_id <- data@meta.data$cells_id[order(data$cells_id, decreasing = F)]`. This is just to avoid possible bugs.

Reply:
> I cannot suggest code which has not been changed. Could you do the same thing that I suggest in the previous comment?
> 
> Another possibility is to reorder the cells in line 40 `result$cells_id <- data@meta.data$cells_id[order(data$cells_id, decreasing = F)]`. This is just to avoid possible bugs.

I agree, I like this one better I think